### PR TITLE
Add BERT-large config to language modelling README

### PR DESCRIPTION
**Pull request by sergiopperez**
# What does this PR do?

Updates README in [optimum-graphcore/examples/language-modeling/](https://github.com/huggingface/optimum-graphcore/tree/main/examples/language-modeling) to include BERT-large commands.

Checkpoints produced with this commands have been uploaded to [Graphcore/bert-large-uncased](https://huggingface.co/Graphcore/bert-large-uncased). The final loss after phase 2 was 1.2 and the SQuAD fine-tuning scores were EM 84.87 and F1 91.14 (reference is 84.1 and 90.9 resp. from Devlin et al. 2018)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Review by jimypbr — APPROVED** [review]
thanks!

File: examples/language-modeling/README.md [hunk 1]
```diff
@@ -96,7 +97,66 @@ python examples/language-modeling/run_pretraining.py \
...
+  --logging_steps 5 \
+  --max_seq_length 512 \
+  --max_steps 2038 \
+  --is_already_preprocessed \
+  --dataloader_num_workers 96 \
+  --dataloader_mode async_rebatched \
+  --lamb \
+  --lamb_no_bias_correction \
+  --per_device_train_batch_size 2 \
+  --gradient_accumulation_steps 512 \
```
Comment (jimypbr) [inline examples/language-modeling/README.md:149]: change to 2048 with pod16

File: examples/language-modeling/README.md [hunk 2]
```diff
@@ -96,7 +97,66 @@ python examples/language-modeling/run_pretraining.py \
...
+  --logging_steps 5 \
+  --max_seq_length 128 \
+  --max_steps 10500 \
+  --is_already_preprocessed \
+  --dataloader_num_workers 64 \
+  --dataloader_mode async_rebatched \
+  --lamb \
+  --lamb_no_bias_correction \
+  --per_device_train_batch_size 8 \
+  --gradient_accumulation_steps 512 \
```
Comment (jimypbr) [inline examples/language-modeling/README.md:119]: change to 2048 wit pod16

File: examples/language-modeling/README.md [hunk 3]
```diff
@@ -96,7 +97,66 @@ python examples/language-modeling/run_pretraining.py \
...
+  --per_device_train_batch_size 2 \
+  --gradient_accumulation_steps 512 \
+  --pod_type pod64 \
+  --learning_rate 0.002828 \
+  --lr_scheduler_type linear \
+  --loss_scaling 16384 \
+  --weight_decay 0.01 \
+  --warmup_ratio 0.128 \
+  --config_overrides "layer_norm_eps=0.001" \
+  --ipu_config_overrides "matmul_proportion=[0.15 0.2 0.2 0.2]" \
```
Comment (jimypbr) [inline examples/language-modeling/README.md:157]: I needed 0.19 when I tried this. Change to 0.19 from 0.2
